modules/routing/routing.cc

```diff
@@ -144,6 +146,55 @@ double Routing::GetRoutingLength(const RoutingResponse& routing_response) {
   return length;
 }
 
+bool Routing::GetParkingID(const PointENU& parking_point,
+                           std::string* parking_space_id) {
+  // search current parking space id associated with parking point.
+  constexpr double kDistance = 0.01;  // meter
+  std::vector<ParkingSpaceInfoConstPtr> parking_spaces;
+  if (!hdmap_->GetParkingSpaces(parking_point, kDistance, &parking_spaces)) {
+    *parking_space_id = parking_spaces.front()->id().id();
+    return true;
+  }
+  return false;
+}
+
+bool Routing::FillParkingID(RoutingResponse* routing_response) {
+  // set parking space ID when
+  //  has parking info && has parking point && NOT has parking space id && get
+  //  ID successfully
+  const auto& routing_request = routing_response->routing_request();
+  const bool has_parking_info = routing_request.has_parking_info();
+  if (!has_parking_info) {
+    ADEBUG << "routing has no parking info";
+    return false;
+  }
+
+  const bool has_parking_id =
+      routing_request.parking_info().has_parking_space_id();
+  if (has_parking_id) {
+    ADEBUG << "routing already has parking ID";
+    return false;
+  }
+
+  const bool has_parking_point =
+      has_parking_info && routing_request.parking_info().has_parking_point();
+  // not need to extract parking ID when parking_point is not available
+  if (!has_parking_point) {
+    ADEBUG << "routing has no parking point info";
+    return false;
+  }
```

Comment:
consolidate the above 3 checks into one. 
make it less length.

Reply:
Fixed. Thank you.